test/node/object_id_tests.js

```diff
@@ -60,6 +250,12 @@ describe('ObjectId', function () {
     done();
   });
 
+  it('should throw an error if invalid Buffer passed in', function (done) {
+    var a = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]);
+    expect(() => new ObjectId(a)).to.throw(TypeError);
+    done();
```

Comment:
nit: No need to call done, or have it passed in to the test here, done is needed when you are testing something async.